src/main/java/com/cedarsoftware/util/reflect/accessors/AbstractAccessor.java

```diff
@@ -0,0 +1,65 @@
+package com.cedarsoftware.util.reflect.accessors;
+
+import com.cedarsoftware.util.reflect.Accessor;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.Objects;
+
+public abstract class AbstractAccessor implements Accessor {
+
+    private final Field field;
+
+    protected AbstractAccessor(Field field) {
+
+        this.field = field;
+    }
+
+    public Field getField() {
+        return this.field;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+
+        if (o == null || getClass() != o.getClass()) {
```

Comment:
Should these be compared using getClass().getName().equals(o.getClass().getName()) ?

I ask, because I don't know if more than one java.lang.Class instance can be created for the same class - does the JVM/our code manage this to always be one instance, allowing to be compared with identity?  It would be faster to keep it the way you have it, just want to make sure it would never be wrong if multiple instances of Class for the same class can exist.

Reply:
So the JVM spec says:  "Two classes are the same class (and consequently the same type) if they are loaded by the same class loader and they have the same fully qualified name", But if you think about all the HashMaps that were already in json-io previous versions with a class as the key - they were all using a Java - object equals.

    public boolean equals(Object obj) {
        return (this == obj);
    }

Reply:
We should be safe assuming that a ClassLoader only holds onto one Class per fully qualified ClassName